docs-conceptual/azps-11.6.0/authenticate-interactive.md

```diff
@@ -22,55 +22,127 @@ principals or other noninteractive authentication methods.
 
 ## Interactive login
 
-Azure PowerShell's default login authentication method uses a web browser and access token to sign
-in.
+To sign in interactively, use the `Connect-AzAccount` cmdlet. Beginning with Az PowerShell module
+version 12.0.0, Windows systems use Web Account Manager (WAM), and Linux and macOS systems use
+browser-based login by default.
 
-1. To sign in interactively, use the `Connect-AzAccount` cmdlet, which uses an interactive
-   browser-based login prompt by default.
+```azurepowershell
+Connect-AzAccount
+```
+
+- To learn more about WAM, see [Web Account Manager (WAM)](#web-account-manager-wam)
+- To learn more about browser-based login, see [Browser-based login](#browser-based-login)
+
+## Login experience
+
+Beginning with Az PowerShell module version 12.0.0, if you have access to multiple subscriptions,
+you're prompted to select an Azure subscription to login with, as shown in the following example.
+
+```Output
+Please select the account you want to login with.
+
+Retrieving subscriptions for the selection...
+WARNING: To override which subscription Connect-AzAccount selects by default, use `Update-AzConfig -DefaultSubscriptionForLogin 00000000-0000-0000-0000-000000000000`. Go to https://go.microsoft.com/fwlink/?linkid=2200610 for more information.
+[Tenant and subscription selection]
+
+No    Subscription name                     Subscription ID                           Tenant name
+----  ------------------------------------  ----------------------------------------  --------------
+[1]   Facility Services Subscription        00000000-0000-0000-0000-000000000000      Contoso
+[2] * Finance Department Subscription       00000000-0000-0000-0000-000000000000      Contoso
+[3]   Human Resources Subscription          00000000-0000-0000-0000-000000000000      Contoso
+[4]   Information Technology Subscription   00000000-0000-0000-0000-000000000000      Contoso
+
+The default is marked with an *; the default tenant is 'Contoso' and subscription is 'Finance Department Subscription (00000000-0000-0000-0000-000000000000)'.
```

Comment:
- If user is in day 0, there is no default subscription selected. Try Connect-AzAccount after Clear-AzContext to mock day 0 scenario. If user is in day N, Connect-AzAccounts will select last used subscription as default. Is there a way to demonstrate these two scenarios clearly here?
- We have a config to turn off interactive selection and switch back to previous login way, which is Update-AzConfig -LoginExperienceV2 Off, please add this in docs.

Reply:
Very helpful information for our customers. I've incorporated it into the doc.